zcash_client_sqlite/src/wallet.rs

```diff
@@ -143,6 +150,15 @@ pub(crate) mod transparent;
 
 pub(crate) const BLOCK_SAPLING_FRONTIER_ABSENT: &[u8] = &[0x0];
 
+/// A constant for use in converting Unix timestamps to shielded-only diversifier indices. The
+/// value here is intended to be added to the current time, in seconds since the epoch, to obtain
+/// an index that is greater than or equal to 2^31. While it would be possible to use indices in
+/// the range 2^31..2^32, we wish to avoid any confusion with indices in the non-hardened BIP 32
+/// child index derivation space.
```

Comment:
```suggestion
/// an index that is greater than or equal to 2^32. While it would be possible to use indices in
/// the range 2^31..2^32, we wish to avoid any confusion with indices in the BIP 32 child
/// index derivation space.
```